### DISCUSSION TITLE
marvelous rating is broken

### DISCUSSION THREAD

**Discussion by ErenKaya31** [DEMOLITIONDON96/Demolition-Engine#12] [Source Q&A] [answered]
@DEMOLITIONDON96 fix your stupid ass rating

**Comment by Blossomical** [comment, marked as answer]
show an image of whats broken and go to issues @ErenKaya31

**Reply by ErenKaya31** [reply]
the marvelous rating is broken when i hit marvelous doesnt show up

**Reply by Theoyeah** [reply]
?

**Reply by ErenKaya31** [reply]
its only shows up on judge counter

**Reply by ThatOneSillyGuyDon** [reply]
this is already fixed in a later update, just haven't distributed a download for it yet, also, Marvelous rating was left there by accident at the time, and god damn, do you have be such a rude ass mf over a simple bug that got resolved already? at least try to be calm about it

**Reply by Theoyeah** [reply]
do you have images of it ?
